edb/server/server.py

```diff
@@ -150,6 +150,10 @@ def __init__(
         self._max_backend_connections = max_backend_connections
         self._compiler_pool = None
         self._compiler_pool_size = compiler_pool_size
+        self._suggested_client_pool_size = max(
+            min(compiler_pool_size, defines.MAX_SUGGESTED_CLIENT_POOL_SIZE),
```

Comment:
Why is client pool size determined by the compiler pool size?  I think it should be a function of the backend connection pool size, since most queries don't go into the compiler.

Reply:
Yeah, it's a copy/paste typo, good catch. Fixed.